vivisect/qt/main.py

```diff
@@ -201,6 +202,113 @@ def addBookmark(self, va, parent=None):
         if ok:
             self.vw.setVaSetRow('Bookmarks', (va, str(bname)))
 
+    def getMemoryWidgets(self):
+        return self.views.get('VQVivMemoryView')
+
+    def getMemWidgetsByName(self, name='viv', firstonly=True):
+        '''
+        Returns a list of Memory View Widgets with the given name.
+        If "firstonly" is True, only return the first one or None(not a list)
+
+        Returns a tuple of (Widget, DockWidget).  The "Widget" is obtained from
+        the DockWidget, but they both have different powers.
+        '''
+        logger.debug("getWindowsByName(%r, firstonly=%r)", name, firstonly)
+        out = []
+
+        for vqDW in self.getMemoryWidgets():
+            w = vqDW.widget()
+            logger.debug("enn: " + w.getEnviNavName())
```

Comment:
`enn:`? If you're going to throw a ton of logging in here, it has to be clear what it's doing, otherwise it's just more noise.

Reply:
killed.